code/game/objects/machinery/computer/intel_computer.dm

```diff
@@ -95,6 +111,8 @@
 			. = TRUE
 		if("start_progressing")
 			printing = TRUE
+			update_minimap_icon()
+			xeno_message("A hostile is attempting to activate the threat at [get_area(src)] we must stop them!", 'sound/voice/alien/talk2.ogg')
```

Comment:
IMO I'd keep the text warning out, so its consistant with disk computers

Reply:
done